core/templates/dev/head/components/forms/schema_editors/SchemaBasedHtmlEditorDirective.js

```diff
@@ -29,19 +29,25 @@ oppia.directive('schemaBasedHtmlEditor', [
         '/components/forms/schema_editors/' +
         'schema_based_html_editor_directive.html'),
       restrict: 'E',
-      link: function($scope, element, $attrs) {
+      link: function(scope, element, attrs) {
+        // A mutation observer to detect changes in the RTE Editor
         var observer = new MutationObserver(function(mutations) {
+          // A condition to detect if mutation involves addition of node by
+          // drag and drop method.
           if (mutations[mutations.length - 1].addedNodes[0]) {
+            // This condition checks if added node is image type.
             if (mutations[mutations.length - 1]
               .addedNodes[0].nodeName === 'IMG'){
-              var imageRteComeponent =
-              mutations[mutations.length - 1].addedNodes[0];
-              imageRteComeponent.className =
-              'oppia-noninteractive-image block-element';
-              imageRteComeponent.click();
+              // Gets the added image node
+              var addedImgNode = mutations[mutations.length - 1]
+                                      .addedNodes[0];
```

Comment:
Better to indent like so:

    var addedImgNode = mutations[
      mutations.length - 1].addedNodes[0];

Same issue above on line 39.

Also @abp7, is the linter supposed to catch issues like this?

Reply:
Done

Reply:
Hi @AllanYangZhou! I don't think we have a rule for breaking after "[" when there exists a single element but we do have a check for the extra indentation in line 43 (Added in PR #4825).  
Also, there are only 65 characters in lines 42 and 43 combined. Therefore we can have this like: 
```javascript
var addedImgNode = mutations[mutations.length - 1].addedNodes[0];
```
@WickedBrat: Is there any specific reason for breaking this into 2 lines?  

(Sorry for the late response!)

Thanks!

Reply:
Actually not! I was showing an error that's why I pushed it to next line :/ . It's fixed now

Reply:
Great, thanks @apb7 !